src/test/java/com/google/step/snippet/W3SchoolClientTest.java

```diff
@@ -0,0 +1,60 @@
+package com.google.step.snippet;
+
+import com.google.step.snippet.data.Card;
+import com.google.step.snippet.external.W3SchoolClient;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public final class W3SchoolClientTest {
+
+  W3SchoolClient client = new W3SchoolClient();
+
+  @Test
+  public void htmlImage() {
```

Comment:
Can you make the test method more descriptive as to what you're testing / write some JavaDoc? I am not sure the rationale behind these particular links.

Reply:
Tried to make the test methods more descriptive. Not sure if additional JavaDoc is needed now?